src/server.c

```diff
@@ -2629,7 +2629,7 @@ void createSharedObjects(void) {
         shared.bulkhdr[j] = createObject(OBJ_STRING,
             sdscatprintf(sdsempty(),"$%d\r\n",j));
     }
-    /* The following two shared objects, minstring and maxstrings, are not
+    /* The following two shared objects, minstring and maxstring, are not
```

Comment:
manual fix? how does the spell checker handle such things? does it recognize it refers to a variable name?

Reply:
manual fix. Also when i read the code. saw it and changed it...

Reply:
but i see your other branch, doesn't have `maxstrings` in the wordlist.
so does the spell checker know to ignore variable names in comments?
does it warn about `maxstrings`? would it have warned about the one with the typo? (i.e. no variable with that name)

Reply:
It will not warn about `maxstrings`

I don't think it is smart enough to check variable names....
 In fact codespell maintain a dict that use to correct the words. 
 https://github.com/codespell-project/codespell/tree/master/codespell_lib/data